src/lakefs_spec/transaction.py

```diff
@@ -102,15 +102,13 @@ def complete(self, commit: bool = True) -> None:
 
         self.fs._intrans = False
 
-    def create_branch(self, repository: str, branch: str, source_branch: str) -> str:
+    def create_branch(self, repository: str, name: str, source_branch: str) -> str:
```

Comment:
Any reason not to add the `exist_ok` flag to the transaction as well? Since this is essentially a wrapper around the client helper, I'd expect the same interface here.

Reply:
Good catch, added in [68908cc](https://github.com/appliedAI-Initiative/lakefs-spec/pull/154/commits/68908cc1cdaf1ced63ed407b83a8b8413676521f).